libraries/license-info.json

```diff
@@ -1,16 +1,64 @@
 {
-  "spark-md5": "MIT",
-  "text-field-edit": "MIT",
-  "scratchblocks": "MIT",
-  "scratch-blocks": "Apache-2.0",
-  "icu-message-formatter": "Apache-2.0",
-  "vue": "MIT",
-  "tinycolor2": "MIT",
-  "chartjs": "MIT",
-  "color-picker-web-component": "MIT",
-  "comlink": "Apache-2.0",
-  "Sora": "OFL",
-  "sortable": "MIT",
-  "fuse": "Apache-2.0",
-  "idb": "ISC"
+  "spark-md5": {
+    "license": "MIT",
+    "year": "2015",
+    "fullname": "André Cruz <amdfcruz@gmail.com>"
+  },
+  "text-field-edit": {
+    "license": "MIT",
+    "year": "",
+    "fullname": "Federico Brigante <me@fregante.com> (https://fregante.com)"
+  },
+  "scratchblocks": {
+    "license": "MIT",
+    "filename": "MIT-scratchblocks"
+  },
+  "scratch-blocks": {
+    "license": "Apache-2.0"
+  },
+  "icu-message-formatter": {
+    "license": "Apache-2.0"
+  },
+  "vue": {
+    "license": "MIT",
+    "year": "2013-2016",
+    "fullname": "Yuxi Evan You"
+  },
+  "tinycolor2": {
+    "license": "MIT",
+    "year": "",
+    "fullname": "Brian Grinstead, http://briangrinstead.com"
+  },
+  "chartjs": {
+    "license": "MIT",
+    "year": "2014-2022",
+    "fullname": "Chart.js Contributors"
+  },
+  "color-picker-web-component": {
+    "license": "MIT",
+    "year": "2020",
+    "fullname": "Wouter Vroege <wouter@woutervroege.nl>"
```

Comment:
this library has been quite modified by me, not sure how licensing works. source is no longer available tho